concordia/models.py

```diff
@@ -367,6 +367,8 @@ class AssetTranscriptionReservation(models.Model):
 
     created_on = models.DateTimeField(editable=False, auto_now_add=True)
     updated_on = models.DateTimeField(auto_now=True)
+    last_reserve_time = models.DateTimeField(auto_now_add=True, blank=True, null=True)
```

Comment:
It's not immediately obvious to me why `last_reserve_time` isn't `updated_on`?

Reply:
I had played around with dirtying the reservation token, which updated the timestamp. Since I added a separate boolean field to track whether the reservation is valid, the extra timestamp isn't needed. I removed it.